openedx/features/course_experience/__init__.py

```diff
@@ -118,3 +120,16 @@ def course_home_url(course_key):
         pass
 
     return home_url
+
+
+def course_progress_url(course_key) -> str:
```

Comment:
So looking at the [README](https://github.com/openedx/edx-platform/blob/master/openedx/features/course_experience/README.rst?plain=1#L12) for this reminded me that we are actually moving away from this app so we should not add new functionality here.  So I think we should add this new function to the learner_home djangoapp in the LMS instead, even thought the other url is coming from course_experience for now.

Also, hopefully then we don't need to do the import inside this function where it is hidden and it can be pulled up to the top of the file this function ends up in.  Maybe https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/learner_home/utils.py is the right home for it for now.

Reply:
Thanks for the detailed response @feanil. Applied the changes as you suggested. Hope everything's fine this time :)